src/script/conversation/ConversationRepository.ts

```diff
@@ -2354,14 +2406,14 @@ export class ConversationRepository {
       });
   }
 
-  _createTextProto(
-    messageId,
-    textMessage,
-    mentionEntities,
-    quoteEntity,
-    linkPreviews,
-    expectsReadConfirmation,
-    legalHoldStatus,
+  private _createTextProto(
+    messageId: string,
+    textMessage: string,
+    mentionEntities: MentionEntity[],
+    quoteEntity: QuoteEntity,
+    linkPreviews: any[],
+    expectsReadConfirmation: boolean,
+    legalHoldStatus: any,
```

Comment:
type for this is `LegalHoldStatus` and can be imported from `@wireapp/protocol-messaging`